src/styles/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  display: flex;
+  width: 100%;
+  max-width: 1200px;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
+  padding: 0 50px;
+  background-color: $color-bg-white;
```

Comment:
Box-shadow property for header is missing. Pay attention to the design
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/02723a35-3f18-4f5a-ae98-125091d282b4)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/b77a960a-c04f-47dd-9dcf-9d0eedf5f8fb)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/f4c1b1c1-80aa-4837-bb4a-fc1f30f9b68e)